indico/modules/rb/schemas.py

```diff
@@ -518,6 +519,7 @@ class SettingsSchema(mm.Schema):
     hide_module_if_unauthorized = fields.Bool()
     tileserver_url = fields.String(validate=validate.URL(schemes={'http', 'https'}), allow_none=True)
     booking_limit = fields.Int(validate=not_empty)
+    booking_reason_required = EnumField(BookingReasonRequiredOptions, required=True)
```

Comment:
no need to set it to required. we always use `partial=True` when loading data using the `SettingsSchema` which means any required field not present in the sent data will be omitted anyway

Reply:
Fixed

Reply:
I actually had to change the validation for a regex when editing, since they both use the same schema.